src/clarity-angular/popover/common/abstract-popover.ts

```diff
@@ -88,6 +88,11 @@ export abstract class AbstractPopover implements AfterViewChecked, OnDestroy {
         return this.ifOpenService.open ? "visible" : "hidden";
```

Comment:
You should just  put the `visibility: hidden` as part of the new CSS class, and drop this extra binding.

Reply:
good idea. thanks